packages/alias/package.json

```
@@ -9,8 +9,8 @@
     "Jacob Smith"
   ],
   "license": "ISC",
-  "main": "./alias.mjs",
-  "types": "./alias.d.mts",
+  "main": "./alias.register.mjs",
+  "types": "./alias.register.d.mts",
```

```suggestion
  "main": "./alias.loader.mjs",
  "types": "./alias.loader.d.mts",
```

So next pr will be that ?

---

No, the next PR will rename `$loaderName.register.mjs` → `$loaderName.mjs` so that the main entry-point continues to be what it was before 🙂

---

Okay perfect I love that